### DISCUSSION TITLE
Alternative to Google Coral

### DISCUSSION THREAD

**Discussion by Shred99** [blakeblackshear/frigate#13390] [Detector Support] [answered]
### Describe the problem you are having

With support for the Intel NCS2 being removed from Frigate, are there any new alternatives to the Coral?

The Coral is getting pretty long in the tooth now and it does not play well with VMware ESXi.  I'm aware that there is a work-around, but it's pretty kludgy.  I use an Intel NCS2 at present, but if I upgrade to 0.14, I'm back to using a CPU based detector.

Surely there must be something new coming down the pipeline - even if it's Google doing a refresh of the Coral?

### Version

0.13

### Frigate config file

```yaml
-
```

### docker-compose file or Docker CLI command

```yaml
-
```

### Relevant Frigate log output

```shell
-
```

### Relevant go2rtc log output

```shell
-
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

OpenVino

### Screenshots of the Frigate UI's System metrics pages

-

**Comment by NickM-27** [comment, marked as answer]
Support for the hailo-8l has been added for frigate 0.15

**Reply by Shred99** [reply]
Thanks NickM.  I'll sit tight on 0.13 for now and wait.  The hailo-8l looks a fair bit more powerful than the Coral.

**Reply by luckylinux** [reply]
Any idea where to buy ? This is definitively (much) more expensive than the Google Coral though. And I wonder if it's any better supported at all ...

Apart from https://pineboards.io/products/ai-bundle-hailo-8l (is this even Legit ???) which is reasonably priced, everything else seems to be in the 180-250 $/EUR Range ...

Except Aliexpress at 140$ if you buy the whole Raspberry Kit (much MORE if you only want the Module), e.g. https://www.aliexpress.com/item/1005007162890873.html.

EDIT 1: 130$ for the Hailo-8L alone - https://aliexpress.com/item/1005007243818592.html

EDIT 2: Sweet - 78 EUR https://www.reichelt.com/at/de/raspberry-pi-ai-kit-fuer-raspberry-pi-5-rasp-ai-kit-p378915.html?r=1

**Reply by partofthething** [reply]
What about those of us not running a raspberry pi? I'm using a small minicomputer and intended to use a Coral, but the one I got was dead on arrival (no messages in lsusb or dmesg) and so was its replacement and now they're out of stock everywhere.

**Reply by NickM-27** [reply]
There are plenty of options such as openvino with igpu, hailo, etc

**Reply by partofthething** [reply]
I should have clarified that I was looking for a USB solution like the Corals I was trying to use. I see that hailo has a M.2 option available but no USB-C.

**Reply by NickM-27** [reply]
Coral is the only USB option currently available as far as I'm aware
